Dockerfile

```diff
@@ -0,0 +1,48 @@
+#Another attempt at dockerizing frontend
+
+
+# Stage 1: Compile and Build angular codebase
+
+# Use official node image as the base image; Andrew's guide says to use node-14.17.3
+FROM node:14.17.3 as builder
+
+# Set the working directory
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
```

Comment:
this is something you might do before we had `npx`, but `npx` exists now, and its purpose is to let you run any command while including any programs from node_modules in the PATH like this. so the line can be taken out cause you're already using npx in line 22 anyway.

Reply:
removed per your suggestion